packages/suite/src/components/suite/modals/ImportTransaction/index.tsx

```diff
@@ -44,7 +48,7 @@ const ImportTransaction = ({ onCancel, decision }: Props) => {
             heading={<Translation id="TR_IMPORT_CSV_MODAL_TITLE" />}
         >
             <ExampleCSV />
-            <DropZone accept="text/csv" onSelect={onCsvSelect} />
+            <DropZone accept=".csv,text/csv" icon="CSV" onSelect={onCsvSelect} />
```

Comment:
probably overlooked mistake `accept=".csv,text/csv"`

Reply:
According to HTML file input specification, this should be valid. It checks extension and mimetype. Other question is whether it should be passed as an array (as you proposed above) or in the exact format for underlying `<input .../>` (as it is right now).